avaframe/in3Utils/cfgUtils.py

```diff
@@ -102,10 +102,14 @@ def getModuleConfig(module, fileOverride='', modInfo=False):
     else:
         raise FileNotFoundError('None of the provided cfg files exist ')
 
-    # Finally read it
-    cfg = compareConfig(iniFile, modName, compare, modInfo)
-
-    return cfg
+    if modInfo:
+        # Finally read it
+        cfg, modDict = compareConfig(iniFile, modName, compare, modInfo)
+        return cfg, modDict
+    else:
+        # Finally read it
+        cfg = compareConfig(iniFile, modName, compare, modInfo)
+        return cfg
```

Comment:
I suggest: 
```suggestion
    # Finally read it
    cfg, modDict = compareConfig(iniFile, modName, compare, modInfo)
    if modInfo:
        return cfg, modDict
    return cfg
```

Reply:
this is not possible as compareConfig only returns the modDict if modInfo is true

Reply:
you could just always return the modDict. This way felixs solution works and is shorter and nicer and if you call this function somewhere else and don't want to use the modDict just do:
cfg, _ = compareConfig(iniFile, modName, compare, modInfo)

Reply:
changed now modDict is always returned and updated the calls where it is not required